src/index.js

```diff
@@ -49,6 +51,8 @@ function variant(hashRaw, hashEncoded, verify) {
 
     hash(...args) {
       const [password, salt, options, cb] = parseArgs(args);
+      let parsedPassword = password;
+      if (typeof password === 'string') { parsedPassword = new Buffer(password); }
```

Comment:
What do you think about moving these two lines to `parseArgs`, so we don't have to repeat it in both `hashRaw` and `hash`?